tensorflow_model_optimization/g3doc/guide/clustering/index.md

```diff
@@ -1,5 +1,7 @@
 # Weight clustering
 
+<sub>Owned by ARM ML Tooling</sub>
```

Comment:
Same as above, use "Arm ML Tooling" for consistency

Reply:
Done.